src/commonMain/resources/templates/shows/copyForReview-manual.glsl

```diff
@@ -0,0 +1,61 @@
+struct FixtureInfo {
+    vec3 position; // z=left/right y=height x=front-back
+    vec3 rotation; // y=left/right yaw, z=pitch (0 is up), x=roll,
+    mat4 transformation;
+};
+
+struct MovingHeadParams {
+    float pan;
+    float tilt;
+    float colorWheel;
+    float dimmer;
+};
+
+#define PI 3.14159265358979323846
+#define DISTANCE_FORWARD 500.0 // Inches
+#define DISTANCE_SIDEWAYS 300.0 // Inches to each side
+
+uniform FixtureInfo fixtureInfo;
+uniform vec2 floorPosition; // @@XyPad
```

Comment:
Nice! Floor works great for headlights mode, where eyes should point at the road ahead.

Thinking about how to aim in other cases, like at the disco balls, which are just a bit below and aft of eye level and off to one side. Maybe consider a few modes for the position, like floor, sky, and something like horizon, which would be a cylinder surrounding us?